src/h2olog/h2olog.cc

```diff
@@ -54,8 +54,7 @@ Optional arguments:
   -t <tracepoint>   A tracepoint, or fully-qualified probe name to trace. Glob
                     patterns can be used; e.g., "quicly:accept", "h2o:*".
   -S <rate>         Enable random sampling per connection (0.0-1.0). Requires
-                    `usdt-selective-tracing: ON` must be specified in the h2o
-                    config file.
+                    use of `usdt-selective-tracing: ON` in the h2o config file.
```

Comment:
Thank you for spotting this. If we are to change the exlanation in alignment with that of `-A`, then maybe something ilke:
```suggestion
                    use of `usdt-selective-tracing`.
```

Alternatively we could say that it "requires `usdt-selective-tracing: ON`" (notice the omission of "use of", I think it sounds strange when we talk about the exact setting rather than naming the setting), but that might be too verbose.